src/service-api/conf/nginx.conf

```diff
@@ -66,5 +66,9 @@ http {
         proxy_pass        http://dps-validation-service:8081$uri$is_args$args;
     }
 
+    location /dpsnotificationservice/ws {
+        proxy_pass        http://dps-notification-service:8080$uri;
```

Comment:
this will change to https in the future?

Reply:
no it will stay, I'm still not sure if you are going to use this file or if this is just a helper to build it for you